bot/skills/roll.py

```diff
@@ -15,10 +19,67 @@
 NUM_BULLETS = 6
 
 
+class DB:
+    """
+        Hussar document:
+        {
+            _id: 420,                                   # int       -- tg user id
+            meta: {...},                                # Dict      -- full tg user object (just in case)
```

Comment:
GDPR states that you can't store user-related data 'just in case', only when you actually need it 🧐

Reply:
This is all the data we are saving from the user. And all of them in public domain
```
        "id": {
            "$numberInt": "567937"
        },
        "first_name": "Egregors",
        "is_bot": false,
        "username": "egregors",
        "language_code": "en"
```

Reply:
Does not matter. It is 'his/her/its' data an 'he/she/it' is endowed with unalienable rights to withdraw them from our possession.

Reply:
sory I dont speack english